modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -170,8 +200,6 @@ private var credentialStore = SynchronizedDictionary<Int, URLCredential>()
             challengeHandlerStore[dataTask.taskIdentifier] = taskDidReceiveChallenge
             credentialStore[dataTask.taskIdentifier] = credential
 
-            dataTask.resume()
```

Comment:
@leszek-s in continuation of the above comment, this change should be reverted.

Reply:
Reverted